webapp/channels/package.json

```diff
@@ -116,6 +116,7 @@
     "@redux-devtools/extension": "3.2.3",
     "@testing-library/jest-dom": "5.16.4",
     "@testing-library/react": "12.1.4",
+    "@testing-library/user-event": "12.1.4",
```

Comment:
Why did this addition not generate new changes in a lock file?

Reply:
I think package lock already have it, as boards is using it.